server/tests/ballot_comparison/test_ballot_comparison.py

```diff
@@ -469,3 +483,49 @@ def test_ballot_comparison_cvr_metadata(
 
     ballot_missing_contest = next(b for b in ballots if b["imprintedId"] == "2-2-4")
     assert ballot_missing_contest["contestsOnBallot"] == [contests[0]["id"]]
+
+
+def test_custom_sample_size_validation(
```

Comment:
test case function names need to be unique - i think that's why you're getting the error in CI

Reply:
not sure if that's the cause of the minerva error - maybe fix this and then let me know if you're still seeing that error too

Reply:
@jonahkagan Fixed this, but still getting the minerva error.